### PR TITLE
WIP: Relnotes 5.5.2

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -5,6 +5,77 @@ This release specifically highlights agility, performance, and manageability and
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-552]
+== Release 5.5.2
+Couchbase Server 5.5.2, released in October 2018, is the second maintenance release in the 5.5.x series for Couchbase Server.
+This maintenance release includes fixes to the index and query services, XDCR, and installer components.
+
+[#fixed-issues-552]
+*Fixed Issues*
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc]: Is there a reason we're not making this an H3?

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -5,6 +5,77 @@ This release specifically highlights agility, performance, and manageability and
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-552]
+== Release 5.5.2
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc:9]: Add a space after heading element.

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,6 +5,77 @@ This release specifically highlights agility, performance, and manageability and
...
+Couchbase Server 5.5.2, released in October 2018, is the second maintenance release in the 5.5.x series for Couchbase Server.
+This maintenance release includes fixes to the index and query services, XDCR, and installer components.
+
+[#fixed-issues-552]
+*Fixed Issues*
+
+*Cross Datacenter Replication(XDCR)*
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc]: Add space before (XDCR)

File: modules/release-notes/pages/relnotes.adoc [hunk 4]
```diff
@@ -5,6 +5,77 @@ This release specifically highlights agility, performance, and manageability and
...
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31141[MB-31141^]
+| Fixed an issue where replication from version 4.5.1 to version 5.5.x caused corruption on deleted documents leading to inability to rebalance.
+|===
+
+*Index Service*
+
+[#table_fixedissues_data_552,cols="25,66"]
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc]: Should this be #table_fixedissues_index_552, not #table_fixedissues_data_552?

**Review by eric-schneider — CHANGES_REQUESTED** [review]
In addition to the specific comments, there is mixed use of starting sentences on a new line. While this isn't actually an issue, it's hard to tell if the intent is to have multiple paragraphs. If the intent is to have new paragraphs, then there needs to be a blank line between sentences.

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc]: Good catch. Cleaning this up as the new site supports H3s now.

File: modules/release-notes/pages/relnotes.adoc (hunk 4 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc]: Fixed

**Review by eric-schneider — APPROVED** [review]

**Comment by eric-schneider** [comment]
lgtm

**Pull request merged**